index.html

```diff
@@ -6,5 +6,54 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Personal website</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>Borislav Borislavov</h1>
+      <section class="topnav">
+        <a href="#contact">Contact</a>
+        <a href="#bio">Biography</a>
```

Comment:
You can put the anchors in a list.
See more here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#examples

Reply:
Put the anchors in a list ! Thanks ! 😃